### PR TITLE
Feature/7758 review tests for stacks webapi

### PR DESCRIPTION
**Pull request by sharonrussell**
#### 📲 What

Adding a load more unit tests to the stacks-webapi repo and increasing test coverage dramatically

#### ✅ Acceptance criteria Checklist

- [ ] Code peer reviewed?
- [ ] Documentation has been updated to reflect the changes?
- [ ] Passing all automated tests, including a successful deployment?
- [ ] Passing any exploratory testing?
- [ ] Rebased/merged with latest changes from development and re-tested?
- [ ] Meeting the Coding Standards?

### CODE REVIEW

File: src/simple-api/src/api/xxENSONOxx.xxSTACKSxx.API/Authentication/JwtBearerAuthenticationConfigurationExtension.cs [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ﻿namespace xxENSONOxx.xxSTACKSxx.API.Authentication;
 
-public class JwtBearerAuthenticationConfiguration
+public class JwtBearerAuthenticationConfigurationExtension
```
Comment (chris31389) [inline src/simple-api/src/api/xxENSONOxx.xxSTACKSxx.API/Authentication/JwtBearerAuthenticationConfigurationExtension.cs:3]: A class name ending with `Extension`?  Was there a problem with the name before?

Comment (sharonrussell) [reply src/simple-api/src/api/xxENSONOxx.xxSTACKSxx.API/Authentication/JwtBearerAuthenticationConfigurationExtension.cs:3]: That's a mistake, will revert

**Review by chris31389 — APPROVED** [review]

File: src/simple-api/src/api/xxENSONOxx.xxSTACKSxx.API/Authorization/ConfigurableAuthorizationPolicyProvider.cs [hunk 2]
```diff
@@ -32,3 +32,5 @@ private Task<AuthorizationPolicy> BuildAuthorizationPolicy(Func<Task<Authorizati
                 : authorizationPolicyFactory();
     }
 }
+
```
Comment (alexanderwjrussell) [inline src/simple-api/src/api/xxENSONOxx.xxSTACKSxx.API/Authorization/ConfigurableAuthorizationPolicyProvider.cs:35]: Might want to have a look at spacing?

File: src/simple-api/src/api/xxENSONOxx.xxSTACKSxx.API/Program.cs [hunk 3]
```diff
@@ -212,3 +213,5 @@ static void AddSwagger(IServiceCollection services, JwtBearerAuthenticationConfi
         c.DocumentFilter<VersionPathFilter>("/v2");
     });
 }
+[ExcludeFromCodeCoverage]
```
Comment (alexanderwjrussell) [inline src/simple-api/src/api/xxENSONOxx.xxSTACKSxx.API/Program.cs:216]: Spacing seems off here

**Pull request merged**